src/njs_parser.c

```diff
@@ -4630,8 +4650,9 @@ njs_parser_statement_wo_node(njs_parser_t *parser, njs_lexer_token_t *token,
         return njs_parser_variable_statement(parser, token, current);
 
     case NJS_TOKEN_SEMICOLON:
-        njs_lexer_consume_token(parser->lexer, 1);
-        return njs_parser_stack_pop(parser);
+        /* Insert empty block statement. */
```

Comment:
A comment should not state what we can infer from the code.
Rather it should answer why question, not what.